Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/IMixedRealityInputSource.cs

```diff
@@ -12,17 +12,41 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem
     /// </summary>
     public interface IMixedRealityInputSource : IEqualityComparer
     {
+        /// <summary>
+        /// The ID assigned to the Controller
+        /// </summary>
         uint SourceId { get; }
 
+        /// <summary>
+        /// The Name assigned to the Controller
```

Comment:
Not event a controller, but the source itself. Remember an input source is more than just a controller, it could be a mouse, or a keyboard as well.

Reply:
Ahh, forgot to update the descriptions after moving them. Will fix that as it'll easily be missed

Reply:
fixed the interface descriptions